src/ArduinoIoTCloudTCP.cpp

```diff
@@ -497,16 +501,13 @@ ArduinoIoTCloudTCP::State ArduinoIoTCloudTCP::handle_CheckDeviceConfig()
     return State::ConnectPhy;
   }
 
-  if(getThingIdOutdatedFlag())
+  if(_deviceSubscribedToThing == true)
   {
-    if(_deviceSubscribedToThing == true)
-    {
-      /* Unsubscribe from old things topics and go oi with a new subsctiption */
-      _mqttClient.unsubscribe(_shadowTopicIn);
-      _mqttClient.unsubscribe(_dataTopicIn);
+    /* Unsubscribe from old things topics and go oi with a new subsctiption */
```

Comment:
Typo in comment, `oi` -> `on` (I guess).

Reply:
yes, it was already fixed here https://github.com/arduino-libraries/ArduinoIoTCloud/pull/297/commits/defe4dd583f8fa16ccd286f7ff63b4e732ec670e i know it has nothig to do with delay macros... i will create a separate commit and squash it to the original one

Reply:
Removed the fix from here https://github.com/arduino-libraries/ArduinoIoTCloud/commit/defe4dd583f8fa16ccd286f7ff63b4e732ec670e and squashed directly here https://github.com/arduino-libraries/ArduinoIoTCloud/pull/297/commits/a0916cdce61fd0ab264358dbff68328eea8eeb56